easybuild/easyblocks/p/petsc.py

```diff
@@ -312,10 +317,18 @@ def sanity_check_step(self):
         else:
             libext = "a"
 
-        custom_paths = {
-                        'files': [os.path.join(prefix2, "lib", "libpetsc.%s" % libext)],
-                        'dirs': [os.path.join(prefix1, "bin"), os.path.join(prefix2, "conf"),
+        if LooseVersion(self.version) < LooseVersion("3.6"):
+            custom_paths = {
+                            'files': [os.path.join(prefix2, "lib", "libpetsc.%s" % libext)],
+                            'dirs': [os.path.join(prefix1, "bin"), os.path.join(prefix2, "conf"),
```

Comment:
hmm, the only thing different here is that the `conf` dir is no longer there in recent versions?

I'm willing to just simply drop that from the sanity check then, since if this is not there, it's not something EB can do much about anyway? As long as the binaries/libraries/header files are there, things should be fine?